scripts/configure_delivery.sh

```diff
@@ -68,30 +68,23 @@ done
 
 # Ops Project
 OPS_ENVIRONMENT_DIR=terraform/environments/ops
+TFVARS_FILE="${OPS_ENVIRONMENT_DIR}/terraform.tfvars"
+TFVARS=$(grep -zvwE "repo_(owner|name)" $TFVARS_FILE)
+(echo "$TFVARS") > $TFVARS_FILE
 cat >> "${OPS_ENVIRONMENT_DIR}/terraform.tfvars" <<EOF
-setup_cd_system="true"
 repo_owner="${REPO_OWNER}"
 repo_name="${REPO_NAME}"
 EOF
 terraform -chdir=${OPS_ENVIRONMENT_DIR} apply --auto-approve
 
 # Staging Project
+export TF_VAR_ops_project_id="$OPS_PROJECT"
```

Comment:
I would advise to using only one method of populating terraform variables. Either `TF_VAR` or `terraform.tfvars` but not both.